### ISSUE TITLE
Issue: Not getting good chat  results on enabling Coversation Memory in langchain

### ISSUE THREAD

**Issue by dkiran1** [langchain-ai/langchain#7921]
### Issue you'd like to raise.

Hi,
Here is how i have initilized conversation memory 
memory = ConversationSummaryBufferMemory(llm=llm_model,memory_key="chat_history",return_messages=True,max_token_limit=500)
Here is how I have used ConversationalRetrievalChain
chain=ConversationalRetrievalChain.from_llm(llm_model,retriever=vector.as_retriever(search_kwargs={"k": 10}), memory=memory,verbose=True)
I could see that answer for first question will be good, on asking further questions with 
result = chain({"question": <question>}) answer from bot is not good for few questions. With verbose enabled I have observed that apart from the actual history langchain is adding  few more conversations by itself. Is there a way to supress this(adding additional conversation)
I also tried with ConversationBufferWindowMemory aswell. There also Iam seeing the performance drop. I tried with new langchain version as well "0.0.235", there also Iam seeing the same issue. Could you help me to undersatnd if this is an existing issue or any mistake am I doing while configuring

**Comment by dkiran1** [comment]
Thanks a lot for your  response, 
As you told, Since the questions are independent of each other, may be the chat response performance might be deteriorating
Regrading temperature, I have already set to 0 to avoid radomness, Also I tried to use from langchain.tokenizer import TikTok
But Iam getting "ModuleNotFoundError: No module named 'langchain.tokenizer'" error, I tried with 0.0.235 version also, Also I have installed tiktoken as well. Is there any thing extra needs to be installed? Also is there any contorl that we can put on standalone question that it generates?
